science/digital_rf/Portfile

```diff
@@ -5,8 +5,8 @@ PortGroup           cmake 1.1
 PortGroup           github 1.0
 PortGroup           cxx11 1.1
 
-github.setup        MITHaystack digital_rf 2.6.0
-revision            1
+github.setup        MITHaystack digital_rf 2.6.1
+revision            2
```

Comment:
The `revision` line should be deleted because it starts at 0 when increasing the version and that is the default value.